### DISCUSSION TITLE
Is there any plan for adding quiver plot to altair?

### DISCUSSION THREAD

**Discussion by yeoloon** [vega/altair#2506] [General]
For those people who work on vector data, like ocean currents, wind velocities, quiver plot is a very important way for displaying the data. Nonetheless, it seems that except for matplotlib there are no simple ways to make such plots. It will be great if Altair can add a new mark  property for quiver plot.

**Comment by jakevdp** [comment]
Not that I am aware of - there is a years-old feature request for this in Vega-Lite: https://github.com/vega/vega-lite/issues/4270

**Comment by joelostblom** [comment]
@yeoloon  There are a few way you might be able to approximate this:

- Using a wedge-shaped point mark https://vega.github.io/vega-lite/examples/point_angle_windvector.html
- Combining a line/rule with a point mark https://github.com/queryverse/VegaLite.jl/issues/130
- Using a trail mark https://altair-viz.github.io/gallery/comet_chart.html#comet-chart-example

They would all require translation into altair or a little bit of fiddling, so if you create an example based on one of those, feel free to open a PR to add it to the example gallery.
